README.md

```diff
@@ -52,11 +52,11 @@ I feel like this could be cleaner...
 
 TODO: Document.
 
-```yaml
-roaster:
-  type: Behmor 1600+
-  timer: 20:30
-  parameters: P1, A
+```toml
+[roaster]
+	type       = "Behmor 1600+"
+	timer      = "20:30"
+	parameters = ["P1", "A"]
```

Comment:
Not sure honestly, seemed like what we want, but could make validation harder. That said, if we're going to support different roaster models, it could be a more simple approach than having a bunch of required and optional members of the object. Thoughts?